[ADD] 14.0 account_move_cutoff: deferred revenu/expenses (#256)
OCA/account-closing: account_move_cutoff/README.rst

   !! This file is generated by oca-gen-addon-readme !!
   !! changes will be overwritten.                   !!
   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
    :target: https://odoo-community.org/page/development-status
    :alt: Beta
.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
    :alt: License: AGPL-3
.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Faccount--closing-lightgray.png?logo=github
    :target: https://github.com/OCA/account-closing/tree/14.0/account_move_cutoff
    :alt: OCA/account-closing
.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
    :target: https://translation.odoo-community.org/projects/account-closing-14-0/account-closing-14-0-account_move_cutoff
    :alt: Translate me on Weblate
.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
    :target: https://runbot.odoo-community.org/runbot/89/14.0
    :alt: Try me on Runbot

|badge1| |badge2| |badge3| |badge4| |badge5| 

> This module allows to generate cu-toff entries automatically when posting former

Reviewer: ```suggestion
This module allows to generate cut-off entries automatically when posting former
```

entries.

This module is based on `account_invoice_start_end_dates`
which allows to define start end end dates on invoice line (`account.move.line`).

Following assumption have been made before developing this module::

  - New method to compute cutoff amounts can be add by business modules

.. note::

>     This module as been developed with some opinionated design
>     do not depends on `account_cutoff_base`. Because::

Reviewer: ```suggestion
    This module has few design choices that require explanation::
```

illustrate the definitions. Consider a sales invoice that is posted on January
16th for a service that spans from the 8th to the 15th of March. So, there are
24 days in January, 1 full month in February, and 15 days in March. The product
is sold for 1000 for a month, so the invoice line amount (excluding VAT) is
calculated as follows::

    24/31 * 1000 + 1000 + 15/31 * 1000 = 2258.06

* **monthly_prorata_temporis** (the default if not set): This method splits amounts
  over the rate of the month the product has been used. The results would be as
  follows:

  - January: **774.19** (`2258.06 - 1000 - 483.87`) (Subtraction is used here to avoid
    rounding discrepancies.)
  - February: **1000.00** (`1 * 2258.06 / (24/31 + 1 + 15/31)`)
  - March: **483.87** (`15/31 * 2258.06 / (24/31 + 1 + 15/31)`)

* **equal**: With this method, the same amount is split over the months of service.

  - January: **752.68** (`2258.06 - 752.69 - 752.69`)
  - February: **752.69** (2258.06 / 3)
  - March: **752.69** (2258.06 / 3)

Please note that this information is subject to change based on updates to the
module. Always refer to the latest documentation for accurate details.

Usage
=====

> 

Reviewer: ```suggestion
```
